src/platform/testing/unit/mocha-setup.js

```diff
@@ -20,6 +20,12 @@ global.__MEGAMENU_CONFIG__ = null;
 chai.use(chaiAsPromised);
 chai.use(chaiDOM);
 
+function filterStackTrace(trace) {
+  return trace
+    .split('\n')
+    .filter(line => !line.includes('node_modules'))
```

Comment:
There's a bunch of JSDom, React, and regenerator noise in stack traces that is never helpful, this removes it.